morph_tool/nrnhines.py

```diff
@@ -278,7 +278,19 @@ class NestedPool(multiprocessing.pool.Pool):  # pylint: disable=abstract-method
 
     class Process(multiprocessing.Process):
         """Class that represents a non-daemon process"""
-        daemon = False
+
+        def __init__(self, *args, **kwargs):
+            """Ensures group=None."""
+            kwargs.pop("group", None)
+            super().__init__(None, *args[1:], **kwargs)
+
+        @property
```

Comment:
A question. If `daemon` argument was passed to constructor, what would happen here?

Reply:
No idea :-)
I think it would just try to set the `deamon` attribute to True, which would be intercepted by the setter.

Reply:
this needs to be verified otherwise why having it

Reply:
this stuff is a bit magical, and is used in several places at bbp (bluepymm, etc...). I wouldn't bother looking at these details tbh.

Reply:
I tried with the `daemon` argument and the test `test_nrnhines.test_isolate` fails. It seems that only this way works.